### DISCUSSION TITLE
HIP-751: Lockable Fractional Amounts of Fungible Tokens on Hedera

### DISCUSSION THREAD

**Discussion by sam-at-luther** [hiero-ledger/hiero-improvement-proposals#759] [Edits]
Starting a thread here to discuss [HIP-751](https://github.com/hashgraph/hedera-improvement-proposal/pull/751)

**Comment by Stephanie-Yi** [comment]
Please migrate discussion to https://github.com/hashgraph/hedera-improvement-proposal/discussions/797.
